cosmogrb/sampler/source.py

```diff
@@ -437,6 +446,8 @@ def sample_photons(self, times):
                 self._source_function.emax,
                 self._source_function.response.effective_area_max,
             )
+        
+        return self._propagate_photons(source_frame_photons)
```

Comment:
nice work. Have you tested the functionality to make sure it works?

Reply:
looking at the plots of 
lightcurve.display_count_spectrum_source and
lightcurve.display_source
it seems to work.
unittests also run fine of course